server/src/main/java/org/opensearch/gateway/remote/ClusterStateDiffManifest.java

```diff
@@ -237,6 +243,9 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             builder.value(index);
         }
         builder.endArray();
+        if (indicesRoutingDiffPath != null) {
+            builder.field(DIFF_FIELD, indicesRoutingDiffPath);
+        }
```

Comment:
Do we need Codec check here?

Reply:
If we are calculating `indicesRoutingDiffPath` we would be on Codec3+.